pydbus/strong_typing.py

```diff
@@ -34,3 +35,10 @@ def decorate(func):
 		verify_arguments(func)
 		return func
 	return decorate
+
+
+class TypedSignal(signal):
+
+	def __init__(self, arguments):
```

Comment:
Well it actually needs to be a `(name, type)` tuple, so in varargs it still would be `typed_signal(("arg1", "s"), ("arg2", "i"))`. And kwargs won't work as the order is not defined. Also I actually didn't use varargs to be in line with `typed_method`, which doesn't use them.